samples/wifi/twt/README.rst

```diff
@@ -31,13 +31,17 @@ When the traffic generator module is used with this sample, the traffic generato
 The server source is available in the :file:`scripts/traffic_gen_server` file.
 This is a `python3` based server, so you must make sure that `python3` is installed on your PC.
 
-To start the server, run the following command:
+To start the server:
 
-.. code-block:: console
+1. Set the :kconfig:option:`CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR` Kconfig option to the IPv4 address of the host running the traffic generator server.
```

Comment:
Should we clarify that this is for the client side i.e., for Zephyr build? or is it implied?

Reply:
I think this is implied well @kittydepa